packages/hop-node/src/watchers/PolygonBridgeWatcher.ts

```diff
@@ -1,19 +1,22 @@
 import BaseWatcher from './classes/BaseWatcher'
 import Logger from 'src/logger'
-import Web3 from 'web3'
 import chainSlugToId from 'src/utils/chainSlugToId'
-import erc20Abi from '@hop-protocol/core/abi/generated/ERC20.json'
 import fetch from 'node-fetch'
 import wait from 'src/utils/wait'
 import wallets from 'src/wallets'
 import { Chain } from 'src/constants'
-import { Contract, Wallet, constants, providers } from 'ethers'
-import { Event } from 'src/types'
+import { FxPortalClient } from '@fxportal/maticjs-fxportal'
 import { L1Bridge as L1BridgeContract } from '@hop-protocol/core/contracts/L1Bridge'
 import { L1ERC20Bridge as L1ERC20BridgeContract } from '@hop-protocol/core/contracts/L1ERC20Bridge'
 import { L2Bridge as L2BridgeContract } from '@hop-protocol/core/contracts/L2Bridge'
-import { MaticPOSClient } from '@maticnetwork/maticjs'
+import { Wallet, providers } from 'ethers'
+import { Web3ClientPlugin } from '@maticnetwork/maticjs-ethers'
 import { config as globalConfig } from 'src/config'
+import { setProofApi, use } from '@maticnetwork/maticjs'
+
+use(Web3ClientPlugin)
+setProofApi('https://apis.matic.network')
```

Comment:
Does it make sense to move the `use()` and `setProofApi` into the constructor so they are more logically aligned within the class? I might be misunderstanding their intended usage.

Reply:
Good idea

Resolved with https://github.com/hop-protocol/hop/pull/384/commits/fef0c0adff680b369340f9b219ca3b5acbd002b8